src/core/docs/debug_capabilities.md

```diff
@@ -0,0 +1,11 @@
+# OpenVINO Core debug capabilities
+
+OpenVINO Core contains a set of different debug capabilities which allow to make developer live a little bit easier.
```

Comment:
allow to make ---> allow making
 live  ---> life 
 overall pretty useless sentence, can we elaborate on how exactly it makes it easier or just remove it?

Reply:
Changed sentence.